tests/unit/h/security/policy/_api_test.py

```diff
@@ -0,0 +1,114 @@
+from unittest.mock import create_autospec, sentinel
+
+import pytest
+from pyramid.request import Request
+
+from h.security.identity import Identity
+from h.security.policy._api import APIPolicy, applicable_policies
+
+
+class TestAPIPolicy:
+    def test_forget(self, api_policy, pyramid_request):
+        assert api_policy.forget(pyramid_request, foo="bar") == []
+
+    def test_identity_when_first_sub_policy_returns_truthy(
+        self, applicable_policies, api_policy, pyramid_request
+    ):
+        applicable_policies.return_value[0].identity.return_value = True
+        applicable_policies.return_value[1].identity.return_value = False
+
+        identity = api_policy.identity(pyramid_request)
+
+        applicable_policies.assert_called_once_with(
+            pyramid_request, sentinel.sub_policies
+        )
+        applicable_policies.return_value[0].identity.assert_called_once_with(
+            pyramid_request
+        )
+        applicable_policies.return_value[1].identity.assert_not_called()
+        assert identity == applicable_policies.return_value[0].identity.return_value
+
+    def test_identity_when_second_sub_policy_returns_truthy(
+        self, applicable_policies, api_policy, pyramid_request
+    ):
+        applicable_policies.return_value[0].identity.return_value = False
+        applicable_policies.return_value[1].identity.return_value = True
+
+        identity = api_policy.identity(pyramid_request)
+
+        applicable_policies.return_value[1].identity.assert_called_once_with(
+            pyramid_request
+        )
+        assert identity == applicable_policies.return_value[1].identity.return_value
+
+    def test_identity_when_all_sub_policies_return_falsey(
+        self, applicable_policies, api_policy, pyramid_request
+    ):
+        applicable_policies.return_value[0].identity.return_value = False
+        applicable_policies.return_value[1].identity.return_value = False
+
+        identity = api_policy.identity(pyramid_request)
+
+        assert identity is None
+
+    def test_identity_when_there_are_no_applicable_policies(
+        self, applicable_policies, api_policy, pyramid_request
+    ):
+        applicable_policies.return_value = []
+
+        identity = api_policy.identity(pyramid_request)
+
+        assert identity is None
```

Comment:
I think it's difficult to come up with clear and concise names for the three parameters you'd need to parametrize: the return values of the `identity()` methods of the first and second applicable sub-policies, and which applicable policy's `identity()` result should be returned. So while you could condense three or four tests into one and save some duplication and some lines of code, this would be at the cost of clarity.